sunpy/database/database.py

```diff
@@ -113,6 +113,40 @@ def __str__(self):  # pragma: no cover
         return errmsg.format(self.database_entry, self.tag_name)
 
 
+def split_database(source_database, destination_database, *query_string):
+    """
+    Queries the source database with the query string, and moves the
+    matched entries to the destination database. When this function is
+    called, the undo feature is disabled for both databases.
+
+    Parameters
+    ----------
+    source_database : Database object of the database on which query is to be made
+    destination_database : Database object of the database to which the matched entries
+                            will be appended
+    query_string : Can consist of multiple arguments. All of them will be ANDed together
+                    to form the final query.
+
```

Comment:
I'm not sure if the formatting in the "parameters" section is correct. I think the indentation must align among each parameter.